src/terminal.about

```diff
@@ -1,5 +1,5 @@
 [About]
-Authors=Abel McClendon <hcmmac%hosscomm.com>;Adam Erdman <hekel%archlinux.info>;Alexander von Gluck IV <kallisti5%unixzen.com>;Alexei Sorokin <sor.alexei%meowr.ru>;Allan Nordhøy <epost%anotheragency.no>;Behdad Esfahbod <behdad%gnome.org>;Christian Persch <chpe%gnome.org>;Christopher Fujino <christopherfujino%gmail.com>;Clement Lefebvre <clement.lefebvre%linuxmint.com>;Friedel Wolff <friedel%translate.org.za>;Guilherme de S. Pastore <gpastore%gnome.org>;Havoc Pennington <hp%redhat.com>;JP Cimalando <jp-dev%inbox.ru>;Laszlo Boros <iamsemmu%gmail.com>;Marcel Dijkstra <marcel.dykstra%gmail.com>;Mariano Suárez-Alvarez <mariano%gnome.org>;Mark Thomas <markbt%efaref.net>;Martin Wimpress <martin%mate-desktop.org>;Michael Terry <michael.terry%canonical.com>;Mike Gabriel <mike.gabriel%das-netzwerkteam.de>;Nikolay Martynov <mar.kolya%gmail.com>;Pablo Barciela <scow%riseup.net>;Perberos <perberos%gmail.com>;Piotr Drąg <piotrdrag%gmail.com>;Scott Balneaves <sbalneav%mate-desktop.org>;Sorokin Alexei <sor.alexei%meowr.ru>;Stefano Karapetsas <stefano%karapetsas.com>;Stephen Krauth <sk42%stephenk.com>;Steve Zesch <stevezesch2%gmail.com>;Vlad Orlov <monsta%inbox.ru>;Wolfgang Ulbrich <mate%raveit.de>;ZenWalker <scow%riseup.net>;
+Authors=Abel McClendon <hcmmac%hosscomm.com>;Adam Erdman <hekel%archlinux.info>;Alexander von Gluck IV <kallisti5%unixzen.com>;Alexei Sorokin <sor.alexei%meowr.ru>;Allan Nordhøy <epost%anotheragency.no>;Behdad Esfahbod <behdad%gnome.org>;Christian Persch <chpe%gnome.org>;Christopher Fujino <christopherfujino%gmail.com>;Clement Lefebvre <clement.lefebvre%linuxmint.com>;Friedel Wolff <friedel%translate.org.za>;Guilherme de S. Pastore <gpastore%gnome.org>;Havoc Pennington <hp%redhat.com>;JP Cimalando <jp-dev%inbox.ru>;Laszlo Boros <iamsemmu%gmail.com>;Laurent Napias <tamplan%free.fr>;Madars Vitolins <madars.vitolins%gmail.com>;Marcel Dijkstra <marcel.dykstra%gmail.com>;Mariano Suárez-Alvarez <mariano%gnome.org>;Mark Thomas <markbt%efaref.net>;Martin Wimpress <martin%mate-desktop.org>;Michael Terry <michael.terry%canonical.com>;Mike Gabriel <mike.gabriel%das-netzwerkteam.de>;Nikolay Martynov <mar.kolya%gmail.com>;Pablo Barciela <scow%riseup.net>;Perberos <perberos%gmail.com>;Piotr Drąg <piotrdrag%gmail.com>;Robert Buj <robert.buj%gmail.com>;Sander Sweers <infirit%gmail.com>;Scott Balneaves <sbalneav%mate-desktop.org>;Sorokin Alexei <sor.alexei%meowr.ru>;Stefan Tauner <stefan.tauner%gmx.at>;Stefano Karapetsas <stefano%karapetsas.com>;Stephen Krauth <sk42%stephenk.com>;Steve Zesch <stevezesch2%gmail.com>;Victor Kareh <vkareh%vkareh.net>;Vlad Orlov <monsta%inbox.ru>;Wolfgang Ulbrich <mate%raveit.de>;Wu Xiaotian <yetist%gmail.com>;acampbell <acampbell%al.precisiontech.biz>;alxpl <alxpl%users.noreply.github.com>;liuyi92 <liuyi19920905%gmail.com>;micheal65536 <micheal65536%users.noreply.github.com>;noone3 <boris.woerner%gmail.com>;osch <oliver at luced de>;rezso <rezso%rezso.net>;
```

Comment:
`;osch <oliver at luced de>;`

we will accept this mail as valid?

Reply:
@sc0w done